git/homework.py

```diff
@@ -15,23 +15,29 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
     If @first and @second has same value should return True
     In another case should return False
     """
-    pass
+    if first == second:
```

Comment:
you do not need this check, operators like `==, is, in` already returns Bool value

Reply:
According to the task the function must check values of two objects. As far as I know "is" compares objects via theirs memory address. For instance,
d = {'a': 3, 'b':4}, d1= {'a': 3, 'b':4}
d == d1 -> True
but
d is d1 -> False # because these are different objects.
I will change the code according to your suggestion.